payment/views.py

```diff
@@ -107,10 +107,15 @@ def get(self, request):
 
                 payment_status = midtrans_transaction.get("transaction_status")
                 transaction_object = get_object_or_404(Transaction, order_id=order_id)
+                print(transaction_object.status)
```

Comment:
Genuinely asking, is this print statement used for debugging?

Reply:
Yes, I have deleted it now. thanks for pointing out